lib/github_adapter.rb

```diff
@@ -3,15 +3,27 @@
 class GithubAdapter
   cattr_accessor :user, :password
 
-  def resource
-    @resource ||= RestClient::Resource.new('https://api.github.com/orgs/chatto-hub-test2/teams', user, password)
+  def create_team_resource
+    @resource ||= RestClient::Resource.new(
+      'https://api.github.com/orgs/chatto-hub-test2/teams',
+      user, password
+    )
   end
 
   def create_team(name, description)
-    response = resource.post({
+    response = create_team_resource.post({
       name: name,
       description: description
     }.to_json, content_type: 'application/json')
     JSON.parse(response)
   end
+
+  def invite_to_team(team_id, username)
+    resource = RestClient::Resource.new(
+      "https://api.github.com/teams/#{team_id}/memberships/#{username}",
+      user, password
+    )
+    resp = resource.put({}.to_json, content_type: 'application/json')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.